src/common/API/storageAPI.tsx

```diff
@@ -0,0 +1,39 @@
+import axios from "axios";
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export const getQueueNumber = async (network: string): Promise<any> => {
```

Comment:
for example i strongly disagree with using Promise<any> here

Reply:
hmm, in this case if we dont use promise here, is there any suggestion that we could use? also u care refering to the return type right?

Reply:
ah sorry i meant using any here, promise is fine

Reply:
ahhh, haha ok i updated it liao